httpx/_utils.py

```diff
@@ -539,3 +542,32 @@ def __eq__(self, other: typing.Any) -> bool:
 
 def warn_deprecated(message: str) -> None:  # pragma: nocover
     warnings.warn(message, DeprecationWarning, stacklevel=2)
+
+
+@contextmanager
+def ensure_context_manager(
```

Comment:
Note that we'll be able to drop both of these helpers as soon as https://github.com/encode/httpcore/pull/206 is merged and released. :+1: